src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -32,7 +32,8 @@ if ((RAJA_DIR OR UMPIRE_DIR) AND NOT CAMP_DIR)
     message(FATAL_ERROR "CAMP_DIR is required if RAJA_DIR or UMPIRE_DIR is provided.")
 endif()
 
-# Note: Let Umpire find Camp, don't find it ourselves
+# Note: Let Umpire find Camp via camp_DIR, don't find it ourselves
+set(camp_DIR ${CAMP_DIR})
```

Comment:
Oh  man :-(

Reply:
This is the CMake convention! Don't shoot the messenger.

Reply:
I'm going to create a loop that will create every possible variation in capitalization for each TPL.... CaMP_DiR, cAmP_Dir...

Reply:
![image](https://github.com/LLNL/axom/assets/334483/a452d97d-39ec-4f63-8d52-c5dec678ebcc)

Reply:
This frightens me, yet I cannot look away....

Reply:
sounds like another item to add to @white238 's CMake/BLT best practices guide.

Reply:
Yes, we will provide guidance on the best practices and add a line to support all uppercase for backwards compatibility.

Reply:
So all uppercase to support Fortran users? 😄

Reply:
well, the FORTRAN users, yes.

Reply:
> well, the FORTRAN users, yes.

*rim shot*

Reply:
Should we require all Fortran users to start every line in their CMake files in column 7?  Sorry an old joke about Fortran 77.